enterprise/api_client/enterprise_catalog.py

```diff
@@ -128,6 +128,13 @@ def get_content_metadata(self, enterprise_customer, enterprise_catalogs=None):
             endpoint = getattr(self.client, self.GET_CONTENT_METADATA_ENDPOINT.format(catalog_uuid))
             try:
                 response = endpoint.get()
+                results = utils.traverse_pagination(response, endpoint)
+                response = {
+                    'count': len(results),
+                    'next': 'None',
+                    'previous': 'None',
+                    'results': results,
+                }
                 for item in response['results']:
```

Comment:
Do we actually need `response` here?  Can this be shortened to
```python
response = endpoint.get()
for item in utils.traverse_pagination(response, endpoint):
    ...
```

Reply:
yeah good point, will update